internal/dao/pod.go

```diff
@@ -195,6 +196,13 @@ func (p *Pod) TailLogs(ctx context.Context, c LogChan, opts LogOptions) error {
 		opts.SingleContainer = true
 		return tailLogs(ctx, p, c, opts)
 	}
+
+	if defaultContainer := getDefaultLogContainer(po); defaultContainer != "" {
```

Comment:
This call should return string, error and propagate error to call site.

Reply:
I think it would be better to return string, bool to check if annotation was found successfully. I would not propagate the error if annotation is not found, I think it is better just to ignore it and show logs for all containers.